src/main/resources/languages/zh-CN.json

```
@@ -221,16 +221,15 @@
       "description": "重载配置文件和数据"
     },
     "resetConst": {
-      "reset_all": "重置所有角色的命座。",
-      "success": "已重置 %s 的命座，重新登录后生效。",
+      "reset_all": "重置所有角色的命之座。",
+      "success": "已重置 %s 的命之座，重新登录后生效。",
       "description": "重置当前角色的命之座，执行命令后需重新登录以生效"
     },
     "resetShopLimit": {
       "success": "重置完成。",
       "description": "重置指定玩家的商店刷新时间"
     },
     "sendMail": {
-      "give_usage": "🇺🇸Usage: give <player> <itemID|itemName> [amount] [level] [refinement]",
       "user_not_exist": "玩家 '%s' 不存在。",
```

```suggestion
    "sendMail": {
      "give_usage": "🇺🇸Usage: give <player> <itemID|itemName> [amount] [level] [refinement]",
      "user_not_exist": "玩家 '%s' 不存在。",
```

`commands.sendMail.give_usage` isn't unused

---

╭(°A°`)╮